package.json

```diff
@@ -14,12 +14,12 @@
   },
   "scripts": {
     "build": "babel src -d dist --ignore test.js",
+    "danger": "danger-ci",
     "clean": "rimraf dist",
     "lint": "eslint 'src/**/*.js'",
     "prettier": "prettier --write '{src/**/,scripts/**/,}*.js'",
-    "prettier:check": "prettier-check '{src/**/,scripts/**/,}*.js'",
+    "prettier:check": "prettier --list-different '{src/**/,scripts/**/,}*.js'",
```

Comment:
We were using `prettier-check` to make it exit with code 1, but now that we're doing it in danger we don't need that anymore.